app/src/test/java/com/codesoom/assignment/TaskServiceTest.java

```diff
@@ -0,0 +1,248 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.application.TaskService;
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("TaskService 클래스")
+public class TaskServiceTest extends TestHelper {
+    private TaskService service;
+
+    private final Task dummyTask1 = dummyTask("1");
+    private final Task dummyTask2 = dummyTask("2");
+    private final Task dummyTask3 = dummyTask("3");
+
+    private Task subject1() {
+        return service.createTask(dummyTask1);
+    }
+
+    private Task subject2() {
+        return service.createTask(dummyTask2);
+    }
+
+    private Task subject3() {
+        return service.createTask(dummyTask3);
+    }
+
+    @BeforeEach
+    private void setUp() {
+        this.service = new TaskService();
+    }
+
+    @Nested
+    @DisplayName("getTasks 메소드는")
+    class Describe_createTasks {
+
+        @Nested
+        @DisplayName("아무런 task가 등록되지 않았다면")
+        class Context_no_tasks {
+            @Test
+            @DisplayName("빈 리스트를 리턴한다")
+            void it_returns_empty_list() {
+                List<Task> tasks = service.getTasks();
+
+                assertThat(tasks).hasSize(0);
+            }
+        }
+
+        @Nested
+        @DisplayName("1개의 task가 등록된 상황에서")
+        class Context_only_one_task {
+            @BeforeEach
+            void setup() {
+                subject1();
+            }
+
+            @Test
+            @DisplayName("1개의 task가 포함된 리스트를 리턴한다")
+            void it_returns_empty_list() {
+                List<Task> tasks = service.getTasks();
+
+                assertThat(tasks).hasSize(1);
+                assertThat(tasks.get(0).getTitle()).isEqualTo(dummyTask1.getTitle());
+            }
+        }
+
+        @Nested
+        @DisplayName("3개의 task가 등록된 상황에")
```

Comment:
이렇게 해봅시다. 그리고 이 컨텍스트를 "언제나 참"으로 만들려면 뭔가를 더 해줘야 할 지도 모릅니다.
```suggestion
        @DisplayName("3개의 task만 등록되어 있다면")
```

Reply:
오해의 여지를 더 줄일 수 있도록 `@DisplayName()`을 더 정확하게 표현해보았습니다 👍 

[Commit](https://github.com/CodeSoom/spring-week3-assignment-1/pull/76/commits/423231be484c2d5a1f2b01ebc3e7e77501e5b3a6)